edb/edgeql/compiler/func.py

```diff
@@ -200,6 +200,12 @@ def compile_FunctionCall(
     return setgen.ensure_set(fcall, typehint=rtype, path_id=path_id, ctx=ctx)
 
 
+CONDITIONAL_OPS = {
+    'std::IF': {0, 2},
+    'std::??': {1},
+}
```

Comment:
Can you add a comment explaining what's in sets? It's clear from the code below that it's about arguments, but only because I see the full diff.